pages/faq.tsx

```diff
@@ -29,132 +32,212 @@ export default function FAQ() {
+                  <li>
+                    It’s not self-promoting (i.e. sell a product/business) or encouraging illegal
+                    activity
+                  </li>
                   <li>It’s not racist, homophobic, sexist, ableist</li>
                   <li>Does not contain hate speech</li>
                   <li>Does not promote or encourage self-harm</li>
-                  <li>For storytellers who wish to remain anonymous, their content does not have any personally identifiable information.</li>
-                  <li>Stories should not have personal identifying information about other people impacted by COVID-19 without their consent</li>
+                  <li>
+                    For storytellers who wish to remain anonymous, their content does not have any
+                    personally identifiable information.
+                  </li>
+                  <li>
+                    Stories should not have personal identifying information about other people
+                    impacted by COVID-19 without their consent
+                  </li>
                 </ol>
-                
-                <p>A story that does not meet these guidelines will not be shared publicly. If you have any questions or concerns, reach out to <a href="mailto:info@mycovidstory.ca">info@mycovidstory.ca</a>.</p>
+
+                <p>
+                  A story that does not meet these guidelines will not be shared publicly. If you
+                  have any questions or concerns, reach out to{' '}
+                  <a href="mailto:info@mycovidstory.ca">info@mycovidstory.ca</a>.
+                </p>
               </AccordionPanel>
             </AccordionItem>
 
             <AccordionItem>
               <h2>
                 <AccordionButton>
                   <Box flex="1" textAlign="left">
-                    <Heading as="h2" size="sm">Who can access private data?</Heading>
+                    <Heading as="h2" size="sm">
+                      Who can access private data?
+                    </Heading>
                   </Box>
                   <AccordionIcon />
                 </AccordionButton>
               </h2>
               <AccordionPanel pb={4}>
-                You have full control over who your information will be shared with — nobody else will have access to any personally identifiable information. We take your privacy seriously and have taken this into consideration at every step. The site does not use any third-party tracking or analytics tools.
+                You have full control over who your information will be shared with — nobody else
+                will have access to any personally identifiable information. We take your privacy
+                seriously and have taken this into consideration at every step. This site does not
+                use cookies or any third-party tracking tools. We use{' '}
+                <a
```

Comment:
If you use `<Link href='...' isExternal />` the isExternal prop will take care of the target and rel for you!

Reply:
As far as I can tell <Link> is meant for internal links?

Reply:
Then why would they have a prop setting the external target and rel? Either way, not a huge deal :)